man/jointScoreTest.Rd

```diff
@@ -0,0 +1,84 @@
+\name{jointScoreTest}
+\alias{jointScoreTest}
+\title{Perform a joint score test}
+\description{\code{jointScoreTest} is used to perform a joint score test of a set of variants using a null model and a matrix of genotype dosages.}
+\usage{
+jointScoreTest(nullmod, G)
+}
+\arguments{
+  \item{nullmod}{A null model object returned by \code{\link{fitNullModel}}.}
+  \item{G}{A matrix of genotype dosages, where samples are the rows and variants are the columns.}
+}
+\details{
+  \code{jointScoreTest} takes the object returned by \code{\link{fitNullModel}} and performs a joint score test for all variants in the \code{G} matrix using this null model.
+  All effect size and PVE estimates in \code{fixef} are adjusted for (i.e. conditional on) all other variants included in \code{G}.
+
+  The \code{G} matrix must be formatted such that the rows are samples and the columns are variants, and the entries are the dosage for that sample and variant.
+  The matrix must have \code{sample.id} as rownames; this is used to match the genotypes to the null model.
+  Therefore, the same sample identifiers must be used in both \code{nullmod} and \code{G}.
+  \code{G} can contain additional samples and ordering is unimportant as long as all samples from the null model are present; it will be subset and reordered to match the set of samples in the null model.
+
+  Colnames for \code{G} are not required.
+  If present, the column names of \code{G} will be used as the rownames of the \code{fixef} element and the column and rownames of the \code{betaCov} element of the output.
+  \code{fixef} and \code{betaCov} will be in the same order as the columns in \code{G}.
+
+  Missing data in \code{G} are not allowed.
+}
+\value{\code{jointScoreTest} returns a list with the following elements:
+  \item{Stat.joint}{Squared joint score test statistic for all variants in \code{G}.}
+  \item{pval.joint}{p-value associated with the joint score test statistic drawn from a \deqn{\chi^2} distribution with \eqn{n_\mathrm{variants}} degrees of freedom.}
+  \item{pve}{Estimate of the proportion of variance explained by the variants in \code{G}.}
```

Comment:
small tweak to language:

Estimate of the proportion of variance explained jointly by the variants in \code{G}. 

Also, should we call this `pve.joint` to match the Stat and pval?

Reply:
Yep, I think that makes sense -- changed to `pve.joint`.